internal/pkg/cli/app_package.go

```diff
@@ -324,13 +351,13 @@ func BuildAppPackageCmd() *cobra.Command {
 				return fmt.Errorf("error retrieving default session: %w", err)
 			}
 			opts.describer = cloudformation.New(sess)
-			return opts.Validate()
+			return nil
 		}),
 		RunE: runCmdE(func(cmd *cobra.Command, args []string) error {
-			if err := opts.Ask(); err != nil {
+			if err := opts.Validate(); err != nil {
```

Comment:
Should here be changed to `o` as well? To keep the consistency.

Reply:
I changed the method receiver's name to `"o"` because of this guideline: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names

> often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client").

That doesn't apply for the variable name :)